src/templates/_macros/market_button.html

```diff
@@ -27,7 +27,7 @@
           {{ _('Installed') }}
         {% elif installed %}
           {{ _('Open') }}
-        {% elif user.has_installed(app.id) or user.has_purchased(app.id) %}
+        {% elif user.has_installed(app.id) or user.has_purchased(app.id) or app.role == 'langpack' %}
```

Comment:
Ah yeah, subtle difference. `has_installed(manifest_url)` checks with `getInstalled()`, but that's different...

I'll add comments and maybe rename things to clarify. We need both methods.